main_scenes/game_scene.gd

```diff
@@ -134,7 +136,7 @@ func run_event(event_id: String):
 	$GUI/GoButton.visible = true
 
 # Send text to the dialogue box, and play it
-func _send_text(text: String, name: String = "MinionNo1", use_right_sprite: bool = false):
+func _send_text(text: String, text_name: String = "MinionNo1", use_right_sprite: bool = false):
```

Comment:
Change not relevant to MR.

Reply:
It's to stop it shadowing a builtin, so helps stop future bugs. Admitttedly maybe not relevant at this point.